index.html

```diff
@@ -8,10 +8,10 @@
     </head>
     <body>
         <div class="container-fluid">
-            <div class="row">
+        <div class="row center">
                 <div class="col-xs-12 col-sm-10 col-md-8 col-lg-6 col-xl-4">
                     <div class="center">
-                        <h1>Who's Who in Techlahoma?</h1>
+                        <h2>Who's Who in Techlahoma?</h2>
```

Comment:
A page should always have an `h1` for accessibility and sematic reasons. If the font-size needs to be reduced, please do so via CSS.

Reply:
okay